Formula/unp64.rb

```diff
@@ -0,0 +1,16 @@
+class Unp64 < Formula
+  desc "Generic C64 prg unpacker,"
+  homepage "http://iancoog.altervista.org"
+  url "http://iancoog.altervista.org/C/unp64_234.7z"
+  sha256 "86968afaa13b6c17fac7577041d5e3f3cc51cb534d818b5f360fddf41a05eaad"
+
+  def install
+    system "make", "-C", "unp64_234/src", "unp64"
+    bin.install "unp64_234/src/Release/unp64"
```

Comment:
No `make install`? No docs, man page, or other stuff to install?

Reply:
it's binary-only. Many of the tools from that particular community are kind of bare bones.